joblib/numpy_pickle_compat.py

```diff
@@ -0,0 +1,234 @@
+"""Numpy pickle compatibility functions."""
+
+import pickle
+import os
+import sys
```

Comment:
Unused import.

There are quite a few other unused imports in this file and others, would you mind fixing them? This should be highlighted by your favourite editor.